convert/convert_windows.ml

```diff
@@ -396,9 +396,59 @@ let convert (g : G.guestfs) source inspect i_firmware
   and configure_pnputil_install () =
     let fb_script = "@echo off\n\
                      \n\
-                     echo Wait for VirtIO drivers to be installed\n\
-                     %systemroot%\\Sysnative\\PnPutil -i -a \
-                     %systemroot%\\Drivers\\Virtio\\*.inf" in
+                     setlocal EnableDelayedExpansion\n\
+                     set log=%~dp0pnputil.log\n\
+                     set inf_dir=%systemroot%\\Drivers\\Virtio\\\n\
+                     echo Installing drivers from %inf_dir% > \"%log%\"\n\
+                     set FAILED=0\n\
+                     set REBOOT_PENDING=0\n\
+                     \n\
+                     timeout /t 10 /nobreak >nul\n\
```

Comment:
Similarly there's no point making the script silent (all the `>nul` redirections). We most likely want to see the output of those commands, in case things go wrong.